WcaOnRails/app/models/competition.rb

```diff
@@ -76,6 +76,12 @@ class Competition < ApplicationRecord
   scope :confirmed, -> { where.not(confirmed_at: nil) }
   scope :not_confirmed, -> { where(confirmed_at: nil) }
 
+  enum free_entry_status: {
```

Comment:
Great! Also +1 for using an explicit hash and not relying on implicit array ordering.
I'd argue that this field should be called something like `free_guest_entry_status` because it does not affect the competitor entry fee.

Reply:
:o indeed